src/packages/range/range.scss

```
@@ -96,29 +100,52 @@
       display: flex;
+      flex-direction: column;
       align-items: center;
-      justify-content: center;
       user-select: none;
-      font-size: $font-size-s;
-      color: $range-color;
-      text-align: center;
-      vertical-align: center;
-      box-sizing: border-box;
-      // transform: translate3d(0, -100%, 0);
+      // 角标底部距游标 2px
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)